README.md

```diff
@@ -400,6 +400,8 @@ GMs can:
     - Deletes the zone you are watching.
 * **zone_list**
     - Lists all active zones in the server, as well as some of their details.
+* **zone_lights** "on/off"
```

Comment:
`zone_lights` should appear before `zone_list` (commands are sorted alphabetically)